closed/OpenJ9.gmk

```diff
@@ -82,26 +116,62 @@ endif
 	build-openj9-tools \
 	#
 
+
+
+# generated_target_rules_build
```

Comment:
Unless I am missing something, I don't see why the buildjdk requires a different macro when the same tasks are required for the jdk and jre.

Reply:
The buildjdk requires a different macro as it requires the files to be directly copied into its lib/compressed directory. The jdk and jre images uses the java.base module via jlink to get those files. I tried getting it into a single macro but it would complain that it would "overriding recipe for target"  or that there were "multiple target patterns"